src/index.ts

```diff
@@ -89,28 +27,24 @@ export = class OutputDcatUs11 {
     res.set('Content-Type', 'application/json');
 
     try {
-      const hostname = req.hostname;
-      const siteModel = await this.fetchSite(hostname, this.getRequestOptions(portalUrl));
-
-      // Use dcatConfig query param if provided, else default to site's config
-      let dcatConfig = typeof req.query.dcatConfig === 'string'
-        ? this.parseProvidedDcatConfig(req.query.dcatConfig as string)
-        : req.query.dcatConfig;
-
-      if (!dcatConfig) {
-        dcatConfig = _.get(siteModel, 'data.feeds.dcatUS11');
+      const { res: { locals: { feedTemplate } }, app: { locals: { feedTemplateTransforms } } }: {
+        res?: {
+          locals?: {
+            feedTemplate?: any
+          }
+        }
+        app: {
+          locals: {
+            feedTemplateTransforms?: TransformsList
+          }
+        }
+      } = req;
+
+      if (!feedTemplate) {
+        throw new RemoteServerError('DCAT-US 1.1 feed template is not provided.', null, 400);
```

Comment:
You don't want a "RemoteServerError" as it indicates the error originates from some other upstream server.  You'd use this to distinguish an error originating from the hub-search service as opposed to the Koop feeds service.

Reply:
Created Custom error for errors originating at DCAT plugin.